pyston/serializer.py

```diff
@@ -482,20 +436,37 @@ def _get_fieldset(self, obj, extended_fieldset, requested_fieldset, exclude_fiel
     def _get_obj_serialization_name(self, obj):
         return '{}__{}'.format(obj._meta.db_table, obj.pk)
 
-    def _to_python(self, obj, serialization_format, requested_fieldset=None, extended_fieldset=None, detailed=False,
-                   exclude_fields=None, allow_tags=False, direct_serialization=False, serialized_objects=None,
-                   **kwargs):
+    def _obj_to_python(self, obj, serialization_format, requested_fieldset=None, extended_fieldset=None,
+                       exclude_fields=None, allow_tags=False, direct_serialization=False,
+                       serialized_objects=None, **kwargs):
         exclude_fields = [] if exclude_fields is None else exclude_fields
         serialized_objects = set() if serialized_objects is None else set(serialized_objects)
         fieldset = self._get_fieldset(obj, extended_fieldset, requested_fieldset, exclude_fields,
-                                      kwargs.get('via'), detailed, direct_serialization, serialized_objects)
+                                      kwargs.get('via'), direct_serialization, serialized_objects)
         serialized_objects.add(self._get_obj_serialization_name(obj))
         return self._fields_to_python(obj, serialization_format, fieldset, requested_fieldset,
                                       serialized_objects=serialized_objects,
                                       direct_serialization=direct_serialization, **kwargs)
 
-    def _can_transform_to_python(self, thing):
-        return isinstance(thing, Model)
+    def serialize(self, data, serialization_format, **kwargs):
+        if isinstance(data, QuerySet):
```

Comment:
zmizel queryset serializer nesmysl aby existovat to má serializovat už model serializer